text/0000-procedural-vtables.md

```diff
@@ -62,9 +62,26 @@ As an example, consider the function which is what normally generates your metad
 
 ```rust
 #[repr(C)]
-struct Pointer<T> {
-    ptr: *mut T,
-    vtable: &'static VTable<T>,
+struct Pointer {
+    ptr: *mut u8,
+    vtable: &'static VTable,
+}
+
+/// For going from a `&SomeStruct<dyn Trait>` to a field of `SomeStruct`.
+unsafe impl CustomProjection for Pointer {
+    unsafe fn project(ptr: Pointer, offset: usize) -> *mut u8 {
+        ptr.ptr.offset(offset),
+    }
+}
+
+unsafe impl CustomProjectionToUnsized for Pointer {
+    /// For going from `&SomeStruct<dyn Trait>` to the unsized field
+    unsafe fn project_unsized(ptr: Pointer, offset: usize) -> Pointer {
+        Pointer {
```

Comment:
Isn't this basically just returning a copy of `ptr`? Is that what it is supposed to be?

Reply:
oops, thanks!